cypress/integration/index.test.js

```diff
@@ -0,0 +1,30 @@
+describe("Index", () => {
+  it("adds a todo and toast when submitted", () => {
+    cy.visit("http://localhost:3000");
+    cy.get("input").type("say hello");
+    cy.get("form").submit();
+    cy.get("#todos").contains("say hello");
+    cy.contains("New todo added: say hello");
```

Comment:
Agree with the strategy of `cy.contains` in real life when we know that the toast has actually been built and we want to verify its existence in the proper user-facing way.

For this assignment, though, should we do more here to assert that this text appears in a toast and not just in a random div? What is the lift for that? (I don't know the React toast library and how well it plays with cypress -- is there a class or similar we can select on?)

Reply:
Totally doable - they have "stable" (guaranteed not to change across non-major releases) class names so we can select on that.